home/migrations/0042_auto_20220125_1128.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 3.1.14 on 2022-01-25 11:28
+import re
+
+from django.db import migrations
+
+
+def _fix_article_paragraph_image_format(apps, schema_editor):
+    Article = apps.get_model('home', 'Article')
+    format_regex = re.compile('format="(left|right)"')
+    articles = Article.objects.all()
+    for article in articles:
+        for block in article.body.stream_data:
+            if block.get('type') == 'paragraph':
+                block['value'] = format_regex.sub('format="fullwidth"', block.get('value'))
```

Comment:
Am I correct in that you're overwriting the information on whether an image should be left- or right- aligned here by making all images full-width? I'm concerned about this as this sounds like information we might want to retain, especially for smaller images (which would now get stretched across the whole screen). We might want to serve images in a more flexible way in the future, and I don't think we should throw this information away.

Could we simply add the 360 px width tag to the left/right aligned image as well, even if that means on desktop some left/right aligned images will not show at full width anymore as they are served at a smaller quality than the column width?

Reply:
Also @cbunicef

Reply:
Yes, we shouldn't discard the right/left align data during migration.

Reply:
We shouldn't throw away the left/right alignment data. I believe our intended solution here is on the platform codebase, rather than the migration.

If we can make the paragraph block interpret images as `{% image [image] width-360 %}` then I think we end up with an image that will be MAX 360 wide, and will also only show as normally expected.

60x60 image is served as 60x60 and keeps its alignment or full-width
700x700 image is served as 360x360 and keeps its alignment or full-width
Both are resized in browser to be appropriate size based on alignment/full-width and column size.

Reply:
- we can't serve images from paragraph block as `{% image [image] width-360 %}` because they are stored as tags in the database
- changing format (in our case image size) is the recommended way from wagtail

image sizes 
| label       | default | proposed |
|-------------|---------|----------|
| full width  | 800px   | 360px    |
| left align  | 500px   | 260px    |
| right align | 500px   | 260px    |

changes default sizes will only require codebase change, and we wouldn't have to perform any kind of data/schema migration

Reply:
What about if we use 
full width = width-360
left/right align = max-360x5000

Reply:
we first unregister those then register new ones (with proposed image sizes) with the same name